AFQ/api.py

```diff
@@ -839,6 +857,8 @@ def _streamlines(self, row):
                 include_track=True)
             afd.write_json(meta_fname, meta)
             self.log_and_save_trk(sft, streamlines_file)
+            row['timing']['Tractography'] =\
```

Comment:
The key here should be "Tracking" not "Tractography"

Reply:
#395